src/app/shared/learning-object/learning-object.component.ts

```diff
@@ -47,25 +48,12 @@ export class LearningObjectListingComponent implements OnInit, OnChanges {
 
     this.collectionService.getCollections().then(collections => {
       this.collections = new Map(collections.map(c => [c.abvName, c.name] as [string, string]));
+    }).catch(error => {
```

Comment:
Is the intention of this catch statement to only suppress the error?

Reply:
That was extraneous code. Removed.